src/errors.py

```diff
@@ -33,80 +35,102 @@ def __init__(self, tb_frame, tb_lineno, tb_next):
     full_tb = extend_traceback(tb, current_stack(1))
     return t, v, full_tb
 
+
 class RedditLoginFailed(Exception):
     pass
 
+
 class ImgurLoginError(Exception):
     pass
 
+
 class FileAlreadyExistsError(Exception):
     pass
 
+
 class NotADownloadableLinkError(Exception):
     pass
 
+
 class AlbumNotDownloadedCompletely(Exception):
     pass
 
+
 class FileNameTooLong(Exception):
     pass
 
+
 class InvalidRedditLink(Exception):
     pass
 
+
 class ProgramModeError(Exception):
     pass
 
+
 class SearchModeError(Exception):
     pass
 
+
 class RedditorNameError(Exception):
     pass
 
+
 class NoMatchingSubmissionFound(Exception):
     pass
 
+
 class NoPrawSupport(Exception):
     pass
 
+
 class NoRedditSupport(Exception):
     pass
 
+
 class MultiredditNotFound(Exception):
     pass
 
+
 class InsufficientPermission(Exception):
     pass
 
+
 class InvalidSortingType(Exception):
     pass
 
-class FileNotFoundError(Exception):
-    pass
```

Comment:
I assume you removed this error as it is not referenced anywhere?

Reply:
FileNotFoundError is an error that is built into Python, so by defining it there, it was overwriting something that already exists. For an exception, it *probably* doesn't matter too much doing that, but in this case it doesn't seem to be necessary. Still, one of the things I have planned for the refactoring is to remove all of the shadowed functions because they can be confusing